tools/osv-linter/internal/checks/checks.go

```diff
@@ -34,14 +37,19 @@ type CheckDef struct {
 	Check       Check
 }
 
+// CheckConfig defines the configuration for a check.
+type CheckConfig struct {
```

Comment:
nit: should we name this "Config" per https://google.github.io/styleguide/go/decisions#repetition given that the package name is "checks"?

Reply:
Yes, thank you. Fixed in 8686bf2